Formula/zsh-async.rb

```diff
@@ -0,0 +1,28 @@
+class ZshAsync < Formula
+  desc "Perform tasks asynchronously without external tools"
+  homepage "https://github.com/mafredri/zsh-async"
+  url "https://github.com/mafredri/zsh-async/archive/v1.8.5.tar.gz"
+  sha256 "3ba4cbc6f560bf941fe80bee45754317dcc444f5f6114a7ebd40ca04eb20910a"
+  license "MIT"
+
+  bottle :unneeded
+
+  def install
+    cp "async.zsh", "async"
+    zsh_function.install "async"
+    prefix.install Dir["*.zsh"]
+  end
+
+  def caveats
+    <<~EOS
```

Comment:
This should be in the upstream documentation

Reply:
@SMillerDev What do you mean?

Reply:
It's not homebrew specific so it shouldn't be a caveat.

Reply:
@SMillerDev Ok, fixed.